pkg/cmd/pulumi/stack.go

```diff
@@ -61,123 +67,20 @@ func newStackCmd() *cobra.Command {
 				return err
 			}
 
-			if showStackName {
-				fmt.Printf("%s\n", s.Ref().Name())
```

Comment:
Should we audit other places where we display `s.Ref().Name()` and update those to know about `--fully-qualify-stack-names`?

Reply:
I did a quick search of Print statements that use `.Ref().Name()` and found these two:

https://github.com/pulumi/pulumi/blob/b2e63c2de126064cc7a7dfef60af824e60743ce3/pkg/cmd/pulumi/config.go#L554-L556

and 

https://github.com/pulumi/pulumi/blob/b2e63c2de126064cc7a7dfef60af824e60743ce3/pkg/cmd/pulumi/config_env_init.go#L123

They seem like log-style messages, maybe worth swapping to `.String()` but also they don't look too important.